File: web/backend/src/models/orderitem.js
module.exports = (sequelize, DataTypes) => {
  const OrderItem = sequelize.define('OrderItem', {
    order_id: {
      type: DataTypes.INTEGER,
      primaryKey: true,
    },
    dish_id: {
      type: DataTypes.INTEGER,
      primaryKey: true,
    },
    quantity: DataTypes.INTEGER,
  }, {
    tableName: 'order_items',
    timestamps: false,
  });
  return OrderItem;
};

File: old_version/web/backend/src/models/order_item.py
from sqlalchemy import Column, Integer, ForeignKey
from ..config.database import Base

class OrderItem(Base):
    __tablename__ = "order_items"

    order_id = Column(Integer, ForeignKey("orders.id"), primary_key=True)
    dish_id = Column(Integer, ForeignKey("dishes.id"), primary_key=True)
    quantity = Column(Integer)